static/app/views/performance/trends/changeExplorer.tsx

```diff
@@ -0,0 +1,330 @@
+                ? t('Regression Metric')
+                : t('Improvement Metric')
+            }
+            value={trendFunction}
+          />
+          <InfoItem label={t('Start Time')} value={breakpointDate} />
+        </ExplorerContainer>
+      </ExplorerContainer>
+      <GraphPanel data-test-id="pce-graph">
+        <strong>{trendParameter.label + ' (' + trendFunction + ')'}</strong>
+        <ExplorerText color={theme.gray300} margin={'-' + space(3)}>
+          {trendView.statsPeriod
+            ? DEFAULT_RELATIVE_PERIODS[trendView.statsPeriod] ||
+              getTimeString(trendView.statsPeriod)
+            : trendView.start + ' - ' + trendView.end}
+        </ExplorerText>
+        <Chart
+          query={trendView.query}
+          project={trendView.project}
+          environment={trendView.environment}
+          start={trendView.start}
+          end={trendView.end}
+          statsPeriod={trendView.statsPeriod}
+          disableXAxis
+          disableLegend
+          neutralColor
+          {...props}
+        />
+      </GraphPanel>
+      <MetricsChart
+        isLoading={isLoading}
+        location={location}
+        transaction={transaction}
+        trendFunction={trendFunction}
+        trendView={trendView}
+        organization={organization}
+      />
+    </React.Fragment>
+  );
+}
+
+function getTimeString(time: string) {
+  const timeMeasurements = {
+    m: 'minutes',
+    h: 'hours',
+    d: 'days',
+    w: 'weeks',
+  };
+
+  const suffix = time.charAt(time.length - 1);
+  const number = time.slice(0, time.length - 1);
+  const measurement =
+    number === '1'
+      ? timeMeasurements[suffix].slice(0, timeMeasurements[suffix].length - 1)
+      : timeMeasurements[suffix];
+
+  const timestring = number === '1' ? measurement : number + ' ' + measurement;
+  return tct('Last [timestring]', {timestring});
+}
+
+function InfoItem({
+  label,
+  value,
+  margin,
+  float,
+}: {
+  label: string;
+  value: string;
+  float?: boolean;
+  margin?: boolean;
+}) {
+  return (
+    <ExplorerContainer margin={margin} float={float}>
+      <Strong>{label}</Strong>
+      <LargeText>{value}</LargeText>
+    </ExplorerContainer>
+  );
+}
+
+function Header(props: HeaderProps) {
+  const {transaction, trendChangeType} = props;
+
+  const regression = trendChangeType === TrendChangeType.REGRESSION;
+
+  return (
+    <HeaderWrapper data-test-id="pce-header">
+      <FireIcon regression={regression} />
+      <HeaderTextWrapper>
+        <ChangeType regression={regression}>
+          {regression ? t('Ongoing Regression') : t('Ongoing Improvement')}
+        </ChangeType>
+        <TransactionName>{transaction.transaction}</TransactionName>
+      </HeaderTextWrapper>
+    </HeaderWrapper>
+  );
+}
+
+function FireIcon({regression}: {regression: boolean}) {
+  return (
+    <IconWrapper regression={regression}>
+      <IconFire color="white" />
+    </IconWrapper>
+  );
+}
```

Comment:
You can probably remove a couple lines here by just changing `IconWrapper` to be named `FireIcon`, since it takes a regression prop it probably isn't generic anyway.